### PR TITLE
Use fissix in order to support newer python versions

### PR DESCRIPTION
**Pull request by dosas**
https://github.com/pytest-dev/nose2pytest/issues/16

### CODE REVIEW

**Comment by jayvdb** [comment]
Looks like there are two expected strings which need to be adjusted to match what fissix produces.

**Comment by dosas** [comment]
Looks like this is the same problem as described here https://github.com/pytest-dev/nose2pytest/issues/15

**Comment by cclauss** [comment]
@dosas Would you be willing to add the changes from #21 and then I can close #21?  I think it would be easier for maintainers to review and accept the changes in this PR if it included working continuous integration testing.

File: setup.cfg
Comment (cclauss) [inline setup.cfg]:
```suggestion
version = attr: nose2pytest.__version__

```

**Comment by dosas** [comment]
@cclauss Done.

**Comment by cclauss** [comment]
Cool.  Thanks.  Let’s see what happens when a maintainer gives the jobs permission to run.

**Comment by dosas** [comment]
It will probably fail due to https://github.com/pytest-dev/nose2pytest/issues/15 ;)

**Comment by cclauss** [comment]
Please rebase.

**Comment by jayvdb** [comment]
ping @dosas

**Comment by dosas** [comment]
pong

**Comment by schollii** [comment]
Looks like this is still WIP? Also there are conflicts to resolve. Let me know when this is ready for review, I look forward to merging this.

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
@schollii Your review, please.

**Comment by schollii** [comment]
I'm not seeing the tox test suite results proving that everything works fine after the changes, are they somewhere else? Sorry it's been a while since I looked at pytest cicd here. 

Update : I can see "Checks waiting for approval" and a button "Approve and run", that's probably what will run tox but can someone confirm that's what I have to do, I don't want to approve PR until I see this.

**Comment by nicoddemus** [comment]
It needed approval to run the test suite (GH does that for first contributors to avoid crypto-spam).

Just gave approval, CI is running.

**Review by schollii — APPROVED** [review]
Changes look good and all tests pass, so good to go!

Thank you @ich and @dosas for doing this.

**Comment by cclauss** [comment]
Next steps?

**Comment by nicoddemus** [comment]
I guess all that is left is for @schollii to merge it?

**Pull request merged**
